devenv/scripts/deploy_addon_private_cluster.sh

```diff
@@ -0,0 +1,18 @@
+# Load variables from tfstate
+export CLUSTER_CLIENT_ID=$(cat terraform.tfstate | jq '.resources[] | select((.name=="clusteridentity") and .type=="azurerm_user_assigned_identity") | .instances[] | .attributes.client_id')
```

Comment:
Can we manage a local env file with Terraform that contains the exports? Best to avoid parsing the tfstate since it isn't exactly a stable versioned api

Reply:
do you mean we should be writing those values to an external file/reading them from there rather than from the tfstate?